README.md

```diff
@@ -1,2 +1,289 @@
+```
+
+
+
+## Configuration
+
+**1.** Create `playwright.config.ts` file with reportportal configuration:
+```typescript
+  import { PlaywrightTestConfig } from '@playwright/test';
+
+  const RPconfig = {
+    'token': '00000000-0000-0000-0000-000000000000',
+    'endpoint': 'https://your.reportportal.server/api/v1',
+    'project': 'Your project',
+    'launch': 'Playwright test',
+    'attributes': [
+      {
+        'key': 'key',
+        'value': 'value',
+      },
+      {
+        'value': 'value',
+      },
+    ],
+    'description': 'Your launch description',
+  };
+
+  const config: PlaywrightTestConfig = {
+    reporter: [[require.resolve('@reportportal/agent-js-playwright'), RPconfig]],
+    testDir: './tests',
+  };
+  export default config;
+```
+
+| Parameter             | Description                                                                                                       |
+| --------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| token                 | User's Report Portal token from which you want to send requests. It can be found on the profile page of this user.|
+| endpoint              | URL of your server. For example 'https://server:8080/api/v1'.                                                     |
+| launch                | Name of launch at creation.                                                                                       |
+| project               | The name of the project in which the launches will be created.                                                    |
+| rerun                 | *Default: false.* Enable [rerun](https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/rerun.md)|
+| rerunOf               | UUID of launch you want to rerun. If not specified, report portal will update the latest launch with the same name|
+| mode                  | Launch mode. Allowable values *DEFAULT* (by default) or *DEBUG*.|
+| skippedIssue          | *Default: true.* ReportPortal provides feature to mark skipped tests as not 'To Investigate' items on WS side.<br> Parameter could be equal boolean values:<br> *TRUE* - skipped tests considered as issues and will be marked as 'To Investigate' on Report Portal.<br> *FALSE* - skipped tests will not be marked as 'To Investigate' on application.|
+| debug                 | This flag allows seeing the logs of the client-javascript. Useful for debugging.|
+
+
+**2.** Add script to `package.json` file:
+```json
+{
+  "scripts": {
+    "test": "npx playwright test --config=playwright.config.ts"
+  }
+}
+
+```
+
+
+## Reporting
+
+This reporter provides Reporting API to use it directly in tests to send some additional data to the report.
+
+
+To start using the `ReportingApi` in tests, just import it from `'@reportportal/agent-js-playwright'`:
+```javascript
+import { ReportingApi } from '@reportportal/agent-js-playwright/src/reportingApi';
```

Comment:
Are you sure the user should import `ReportingApi` from the sources? It should be imported from `build` directory or just from the lib.
F.e. as in https://github.com/reportportal/agent-js-webdriverio#using-reportingapi